code/modules/antagonists/fugitive/hunters/hunter_gear.dm

```diff
@@ -150,7 +150,7 @@
 
 	AddComponent(/datum/component/bloody_spreader,\
 		blood_left = INFINITY,\
-		blood_dna = list("meaty DNA" = "MT-"),\
+		blood_dna = list("meaty DNA" = BLOOD_TYPE_MEAT),\
```

Comment:
...And less prone to errors (Isn't this supposed to be `list("meaty DNA" = get_blood_type_by_name(BLOOD_TYPE_MEAT))`?)